### PR TITLE
fix backwards compatibility problem with elastic search observers

### PR DESCRIPTION
**Pull request by miiu96**
Fix backwards compatibility problem with observers that index data in elasticsearch database.

### CODE REVIEW

File: core/indexer/elasticClientCommon.go [hunk 1]
```diff
@@ -77,6 +77,10 @@ func elasticDefaultErrorResponseHandler(res *esapi.Response) error {
 		if errIsAlreadyExists(responseBody) {
 			return nil
 		}
+		if errAliasAlreadyExits(responseBody) {
+			log.Debug("alias already exits", "response", fmt.Sprintf("%v", responseBody))
```
Comment (bogdan-rosianu) [inline core/indexer/elasticClientCommon.go:81]: exists*

File: core/indexer/elasticClientCommon.go [hunk 2]
```diff
@@ -109,6 +113,29 @@ func errIsAlreadyExists(response map[string]interface{}) bool {
 	return existsString == alreadyExistsMessage
 }
 
+func errAliasAlreadyExits(response map[string]interface{}) bool {
```
Comment (bogdan-rosianu) [inline core/indexer/elasticClientCommon.go:116]: rename to `isErrAliasAlreadyExists`

File: core/indexer/elasticClientCommon.go [hunk 3]
```diff
@@ -109,6 +113,29 @@ func errIsAlreadyExists(response map[string]interface{}) bool {
...
+	aliasExitsMessage := "invalid_alias_name_exception"
```
Comment (bogdan-rosianu) [inline core/indexer/elasticClientCommon.go:117]: exists*

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: core/indexer/elasticClientCommon.go (hunk 1 above)
Comment (miiu96) [reply core/indexer/elasticClientCommon.go:81]: done.

File: core/indexer/elasticClientCommon.go (hunk 2 above)
Comment (miiu96) [reply core/indexer/elasticClientCommon.go:116]: done.

File: core/indexer/elasticClientCommon.go (hunk 3 above)
Comment (miiu96) [reply core/indexer/elasticClientCommon.go:117]: done.

File: core/indexer/elasticClientCommon.go (hunk 1 above)
Comment (iulianpascalau) [reply core/indexer/elasticClientCommon.go]: you can drop `fmt.Sprintf("%v", ` and pass directly the response body to the log.Debug function call as this internally does a `sprintf("%v")`

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by stefanandy — APPROVED** [review]
System test passed

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**